src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -574,11 +574,23 @@ private protected unsafe TransformerEstimatorSafeHandle CreateTransformerFromEst
                 }
                 else
                 {
-                    fixed (byte* dataRootDir = Encoding.UTF8.GetBytes(AppDomain.CurrentDomain.BaseDirectory + char.MinValue))
+                    byte[] dataRoot;
+
+                    if (Directory.Exists(AppDomain.CurrentDomain.BaseDirectory + "/Data/DateTimeFeaturizer"))
+                    {
```

Comment:
Any idea why this wasn't caught by our unit tests? If there isn't a test for this, can you please add one?

Reply:
This only applies when run through NimbusML. ML.Net itself runs just fine without this. I am not sure how I would add a unit test for that? Any ideas?